blueocean-dashboard/src/main/js/creation/bitbucket/server/BbAddServerDialog.jsx

```diff
@@ -0,0 +1,155 @@
+import React, { PropTypes } from 'react';
+import { observer } from 'mobx-react';
+import debounce from 'lodash.debounce';
+import { Dialog, FormElement, TextInput } from '@jenkins-cd/design-language';
+
+import ServerErrorRenderer from './ServerErrorRenderer';
+
+
+let t = null;
```

Comment:
@i386 yeah, it just makes the code slightly more readable to not preface each translation call with `this.` but could go either way with that.